templates/repo/settings/options.tmpl

```diff
@@ -30,7 +30,7 @@
 				{{if not .Repository.IsFork}}
 					<div class="inline field">
 						<label>{{.i18n.Tr "repo.visibility"}}</label>
-						<div class="ui checkbox">
+						<div class="ui checkbox {{if not .Repository.IsPrivate}} tooltip {{end}}" data-content="{{.i18n.Tr "repo.stars_remove_warning"}}">
```

Comment:
```suggestion
						<div class="ui checkbox{{if not .Repository.IsPrivate}} tooltip{{end}}" data-content="{{.i18n.Tr "repo.stars_remove_warning"}}">
```

Reply:
Also, could you add another condition show it only shows if star count > 0?

Reply:
Would be nice to have imho so we don't confuse users with unnecessary information.